src/System.Security.Cryptography.OpenSsl/tests/project.json

```diff
@@ -30,6 +30,9 @@
     "coreFx.Test.net463": {},
     "coreFx.Test.netcoreapp1.1-ns17": {
       "netstandard1.7": [
+        "win7-x86",
+        "win7-x64",
+        "win10-arm64",
```

Comment:
No, although @joperezr (in another review) recommended adding these back even though they are not used